doc/endianness.md

```diff
@@ -0,0 +1,29 @@
+# Endianness
+
+## default endianness
+
+SPDM 1.0 defines the default endianness is little endian.
+
+libspdm follows that. Assuming the libspdm compiler is using little endian, no swap is required.
+
+## endianness of digital signature
+
+SPDM 1.2 defines the endianness of digital signature for RSA, ECDSA, SM2_DSA, and EdDSA.
+
+libspdm follows that. Because the definition aligns with existing crypto library such as openssl and mbedtls, no swap is required.
+
+SPDM 1.0 and 1.1 are not very clear on the digital signature for RSA and ECDSA.
+
+**libspdm uses big endian** for RSA signature and ECDSA, which algins with SPDM 1.2 and existing crypto library. No swap is required.
```

Comment:
I think we agreed that endianness should be configurable for version 1.0 and 1.1.  Rich Kong will put in a PR.

Reply:
That is future task.
The current document should just describe the current behavior.